### ISSUE TITLE
Health dashboard is broken due to the new olm updates for the bundle format.

### ISSUE THREAD

**Issue by skonto** [openshift-knative/serverless-operator#439]
I am getting this failure when installing from master
```
{"level":"error","ts":"2020-09-10T11:02:09.843Z","logger":"cmd","msg":"Failed to start monitoring","error":"failed to setup monitoring resources: the environment variable \"NAMESPACE\" must be set","stacktrace":"github.com/openshift-knative/serverless-operator/knative-operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/openshift-knative/serverless-operator/knative-operator/vendor/github.com/go-logr/zapr/zapr.go:128\nmain.main\n\t/go/src/github.com/openshift-knative/serverless-operator/knative-operator/cmd/manager/main.go:122\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
```
In https://github.com/openshift-knative/serverless-operator/pull/388 I set in  `olm-catalog/serverless-operator/1.9.0/serverless-operator.v1.9.0.clusterserviceversion.yaml `
```
                    - name: NAMESPACE
                      valueFrom:
                        fieldRef:
                          fieldPath: metadata.namespace
                    - name: DEPLOYMENT_NAME
                      value: "knative-openshift"
```

/cc @markusthoemmes @matzew

**Comment by markusthoemmes** [comment]
Argh. Can you please PR a fix to master and release-1.9? Seems like we should add tests that verify that resources get created properly.

**Comment by skonto** [comment]
yes we need tests. I will create the PRs shortly.
